kernel/packages/unity-interface/UnityInterface.ts

```diff
@@ -90,10 +94,12 @@ export class UnityInterface {
     this.Module = this.gameInstance.Module
     _gameInstance = gameInstance
 
-    if (this.Module) {
-      window.addEventListener('resize', this.resizeCanvasDelayed)
-      this.resizeCanvasDelayed(null)
-      this.waitForFillMouseEventData()
+    if (!EDITOR) {
+      if (this.Module !== undefined) {
```

Comment:
I have just noticed that this line will provoke the null exception that we got before fix it on this [**PR**](https://github.com/decentraland/explorer/pull/1214).

We should create another PR and change this line by this one:
`if (this.Module) {`

Reply:
Thx Santi! I'll do that